docker/run_tests.sh

```diff
@@ -31,8 +31,13 @@ PYTHON="$(which python)"
 
 # Wait for postgres and elasticsearch services to be ready
 urlwait "${DATABASE_URL}" 10
+urlwait "http://${PUBSUB_EMULATOR_HOST}" 10
```

Comment:
This variable is in the environment, but has no protocol since it defines a host and not a url. So, I added the protocol manually.